src/puppet-wechat.ts

```diff
@@ -1145,6 +1146,14 @@ export class PuppetWeChat extends PUPPET.Puppet {
 
     try {
 
+      if (rawPayload.MMStatus === MsgSendStatus.SENDING) {
+        throw new Error('message still sending')
```

Comment:
Instead of throwing an Error at here, can we just wait for the `MsgSendStatus` to be `SUCCESS`?

In order to prevent waiting forever, we can also add a timeout for it, for example, if it does not succeed after 30 seconds, then we will throw an Error.

Reply:
Polling is not a good way, so I removed those checks

Reply:
Good choice, I do not like polling too! :+1: